docker/README.md

```diff
@@ -55,7 +55,7 @@ After this runs, a LightGBM model can be found at `LightGBM-CLI-model.txt`.
 
 For more details on how to configure and use the LightGBM CLI, see https://lightgbm.readthedocs.io/en/latest/Quick-Start.html.
 
-## Running the Python-package Сontainer
+## Running the Python-package Container
```

Comment:
I cannot tell from the view of the diff in GitHub.... what actually changed about this line?

This is what I see:

<img width="296" alt="image" src="https://github.com/user-attachments/assets/2a62eeef-67b5-4eeb-ba95-2b0e2f4b412f">

Was it a Windows-specific line ending character or something? If so I'm fine to make this change, just want to understand.

Reply:
I don't know what the issue was tbh, apparently the issue was the "C". I just retyped it...

Reply:
When you retyped it, did you see this line disappear from the diff? I still see it here.

Reply:
No, retyping made it show up :smile: let me check what the binary representation.

Reply:
Apparently, it's just a weird "C" (https://www.compart.com/en/unicode/U+0421) instead of the normal uppercase "C" (https://www.compart.com/en/unicode/U+0043), no idea where it came from.

Reply:
If you search for "container" in the [README on main](https://raw.githubusercontent.com/microsoft/LightGBM/master/docker/README.md), you currently won't find the occurrences in the headings :smile:

Reply:
haha ok got it! Thanks so much for looking into that.

Reply:
Ahah, it's Cyrillic letter and unfortunately it's located on the same key with Latin letter on for example Russian keyboards.
* But it wasn't me 🥷  (#1008)
😄 

![image](https://github.com/user-attachments/assets/83ac9ac5-6ac9-43af-95da-822c761598f8)